assembl/static2/webpack.config.js

```diff
@@ -54,5 +84,6 @@ module.exports = {
           }
         }),
         new webpack.optimize.UglifyJsPlugin(),
+        new ExtractTextPlugin("[name].css"),
```

Comment:
A) Yes, [name] is a pattern understood by webpack, see http://webpack.github.io/docs/configuration.html#output-filename

B) I'm not familiar at all with hot-module replacement, so I did not do any checks around this. I guess a theme system makes all hot reloading more difficult, that's almost unavoidable I guess.